contracts/handlers/fee/dynamic/TwapNativeTokenFeeHandler.sol

```diff
@@ -21,15 +21,23 @@ contract TwapNativeTokenFeeHandler is TwapFeeHandler {
     /**
         @notice Calculates fee for transaction cost.
         @param destinationDomainID ID of chain deposit will be bridged to.
+        @param depositData Data to be passed to Native Token handler.
         @return fee Returns the fee amount.
         @return tokenAddress Returns the address of the token to be used for fee.
      */
-    function _calculateFee(address, uint8, uint8 destinationDomainID, bytes32, bytes calldata, bytes calldata) internal view override returns (uint256 fee, address tokenAddress) {
+    function _calculateFee(address, uint8, uint8 destinationDomainID, bytes32, bytes calldata depositData, bytes calldata) internal view override returns (uint256 fee, address tokenAddress) {
+        uint256 maxFee = _gasUsed;
```

Comment:
Slightly unrelated to this PR, but can we put the _gasUsed in the TwapFeeHandler constructor and document what it is for.

Reply:
Added into the TwapNativeTokenFeeHandler constructor, as it is not used in the base TwapFeeHandler logic so is not needed for other children.